src/cdomains/valueDomain.ml

```diff
@@ -148,7 +148,8 @@ struct
     match t with
     | t when is_mutex_type t -> `Top
     | TInt (ik,_) -> `Int (ID.top_of ik)
-    | TFloat (FDouble, _) -> `Float (FD.top ()) (* TODO(Practical2022): extend to other floating point types *)
+    | TFloat (FDouble, _) -> `Float (FD.top ())
+    | TFloat (fk, _) -> `Top (* TODO(Practical2022): extend to other floating point types *)
```

Comment:
Why is it `TFloat (fk, _) -> 'Top` here for `init_value`, but lower for the `top_value` function it is `TFloat (fk, _) -> 'Float (FD.top ())` ?
shouldn't that be the same in both cases (`'Top` I assume, since it is still TODO)?